modules/ROOT/pages/configure-acm.adoc

```diff
@@ -302,9 +302,9 @@ Chatter tracking enables your community to create discussions on specific object
 
 === Publish Your Community [[publish-your-community]]
 
-. In the ACM control panel, click *Community Builder*.
+. In the API Community Manager control panel, click *Community Builder*.
 . Click *Publish* on the top right corner, and click *Publish* in the *Publish Your Community* dialog.
-. In a few minutes your community will be published, and you will receive an email notification including your community's public URL. You can also find your community live URL by opening the ACM control panel and clicking *Open Community*.
+. In a few minutes your community will be published, and you will receive an email notification including your community's public URL. You can also find your community live URL by opening the API Community Manager control panel and clicking *Open Community*.
```

Comment:
This is not a step, although you might reword it as one:
". In a few minutes, look for an email notification...URL.
You can also...."

Reply:
Fixed-- thank you!